lib/IRGen/GenDecl.cpp

```diff
@@ -826,14 +826,16 @@ void IRGenModule::addObjCClass(llvm::Constant *classPtr, bool nonlazy) {
     ObjCNonLazyClasses.push_back(classPtr);
 }
 
-void IRGenModule::addRuntimeResolvableType(NominalTypeDecl *nominal) {
+void IRGenModule::addRuntimeResolvableType(GenericTypeDecl *type) {
   // Collect the nominal type records we emit into a special section.
```

Comment:
Can you assert that type is an OpaqueTypeDecl or NominalTypeDecl (and not TypeAliasDecl which is the other subclass of GenericTypeDecl), and fix this comment?

Reply:
We already have that assertion deeper in the pipe, when you try to emit a context descriptor.